assets/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
 sample,file,family_id,paternal_id,maternal_id,sex,phenotype
-sample_1,/path/to/fastq_or_bam/files/sample_1.fastq.gz,FAM,PAT,MAT,1,1
```

Comment:
Would it make sense to have 2 tests samplesheet, one with and one without the sex assigned to test all logical gates? Maybe just a stub run. Just throwing ideas

Reply:
Absolutely! And I have, but you can't see it here.

I removed the test data _and_ samplesheets from this repository, and put it in the [nallo branch of the GMS fork of test-datasets](https://github.com/genomic-medicine-sweden/test-datasets/tree/nallo) to make it more similar to how it's handled in for example raredisease and rnafusion. I made a [PR](https://github.com/genomic-medicine-sweden/test-datasets/pull/1) to update the test data and samplesheets used (the [multisample test](https://github.com/genomic-medicine-sweden/nallo/actions/runs/9092587496/job/24989636313?pr=148) now includes one sample where sex is set to 0). 

But maybe that is problematic, because:
- The changes made are not recorded here
- Updating the test data might break the master test profile

Do you have a suggestion for the best solution here? @jemten?
- I could change the links in master & dev to point to specific commits for each file
- I could move the samplesheets back to this repo, but will it work smoothly when merging dev to master?

I'm sure I will have to continue to refine the test data.

Reply:
I think that it is a good idea to hav the specific commit hash in the url. As you say that way we could have a different sample sheet in master and dev.

Reply:
Might make a patch to master then.